docs/source-pytorch/extensions/callbacks.rst

```diff
@@ -138,6 +138,11 @@ Callback API
 ************
 Here is the full API of methods available in the Callback base class.
 
+.. note::
+
+    Outputs: Hooks that occur on batch end (e.g. ``on_train_batch_end``) will receive any values returned from that step (such as the loss) in the form of a dictionary named ``outputs``.
+    ``outputs["loss"]`` here will be the normalized value w.r.t ``accumulate_grad_batches`` of the loss returned from ``training_step``.
```

Comment:
This is not the right place to put it IMO. 

1. This note is inserted before we explain the Callback: "The Callback class is the base for all the callbacks in Lightning". This seems distracting.
2. I propose to add a simple sentence explaining this in the individual hook documentation. When a user implements the  on_train_batch_end, they would go to the documentation of that hook if they have any question about its signature.

Reply:
@awaelchli, thanks for the suggestion, this change has been pushed and is ready for review.